misc/ansible.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+##
+# Installs ansible via pip
+# param1 - Ansible version to install. Optional, defaults to 2.7.0
+#
+function install_ansible() {
+    version=${1:-2.7.0}
+
+    pip install ansible==${version}
+    echo "Installed ansible==${version}"
+}
+
+##
+# Run ansible-playbook on the node
+# param1 - Playbook to run
+# param2 - Path to inventory file or comma separated list of hosts
+# param3 - Extra variables
+#
+function run_ansible_playbook() {
```

Comment:
This doesn't seem like too much of a value-add. Customers would presumably know how to run ansible playbooks and this wrapper doesn't add much over that.